boards/arm/reel_board/doc/index.rst

```diff
@@ -62,6 +62,20 @@ which can be used as an interrupt source.
 
      reel board back (Credit: PHYTEC)
 
+.. note::
```

Comment:
Instead of a note, I'd make this a subsection, so add a heading and outdent the text.  We can also add a label here so we can reference this section later in the doc:
```
.. _reel_display_v2:

Reel board display update
=========================

Starting summer of 2019, reel boards are equipped with ...

```